src/mutations/CreateCommentMutation.js

```diff
@@ -34,10 +36,15 @@ export default (
     onCompleted: (response, errors) => {
       if (!errors) {
         callback();
-      } else {
-        errorCallback();
       }
+
+      if (errors[0].extensions.proxy.statusCode === '429') {
```

Comment:
this is a quite dangerous way how to access error code. there is no guarantee GraphQL errors have these fields every time.